tests/src/tracing/common/NetperfFile.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+
+namespace Tracing.Tests.Common
+{
+    public class NetperfFile : IDisposable
+    {
+        public string Path { get; }
+        private bool keepOutput { get; }
+
+        private NetperfFile(string fileName, bool keep)
+        {
+            Path = fileName;
+            keepOutput = keep;
+        }
+
+        public void Dispose()
+        {
+            if (keepOutput)
+                Console.WriteLine("\n\tOutput file: {0}", Path);
+            else
+                System.IO.File.Delete(Path);
+        }
+
+        public static NetperfFile Create(string[] args)
+        {
+            if (args.Length >= 1)
+                return new NetperfFile(args[0], true);
```

Comment:
Nit: It'd be nice to name the bool parameters, e.g. `return new NetperfFile(args[0], keep: true);`. Applies below.